server/events/command_runner.go

```diff
@@ -152,6 +152,11 @@ func (c *DefaultCommandRunner) RunCommentCommand(baseRepo models.Repo, maybeHead
 	if !c.validateCtxAndComment(ctx) {
 		return
 	}
+
+	ctx.Mergeable, err = c.VCSClient.PullIsMergeable(baseRepo, pull)
+	if err != nil {
+		ctx.Log.Warn("unable to get commit mergeable status: %s", err)
```

Comment:
```suggestion
                // On error we continue the request with mergeable assumed false.
                // We want to continue because not all apply's will need this status,
                // only if they rely on the mergeability requirement.
                ctx.Mergeable = false
		ctx.Log.Warn("unable to get mergeable status: %s. Continuing with mergeable assumed false", err)
```